### DISCUSSION TITLE
Expandable slider control like in the Windows Store app

### DISCUSSION THREAD

**Discussion by xperiandri** [microsoft/microsoft-ui-xaml#8935] [Q&A]
What is this control? Is there a name of it?
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/2365592/f3e31ab4-319e-402e-92ee-e0d73099ec72)

**Comment by castorix** [comment]
There is a ScrollViewer with 2 Buttons Prev/Next and a PipsPager below
